redis_collections/dicts.py

```diff
@@ -440,7 +442,7 @@ def _update_helper_trans(pipe):
             if isinstance(other, Dict):
                 data.update(other.iteritems(pipe))
             elif isinstance(other, RedisCollection):
-                data.update(other.__iter__(pipe))
+                data.update(collections.Counter(other.__iter__(pipe)))
```

Comment:
This fixes a minor bug / difference in behavior that's now [tested for](https://github.com/honzajavorek/redis-collections/pull/92/files#diff-097d907907f059736cec0843818cdecdR267).